cypress/integration/test.spec.ts

```diff
@@ -23,7 +23,8 @@ describe("test", () => {
   });
 
   it("loads yaml tab", () => {
-    checkTab(/yaml/i, [/api/i, /hello world/i], /hello world/i);
+    checkTab(/yaml/i, [/Introduction/i], /YAML Example/i);
+    checkTab(/yaml/i, [/Hello World/i], /Hello World/i);
```

Comment:
Does this mean untagged pages aren't grouped under API anymore? 

```
- Introduction
- Hello World

vs 

- Introduction
> API
  - Hello World
```

I vaguely remember a generic grouping for untagged items was a requested feature at one point. If this is the case, this could be considered unexpected/breaking change for some people. I personally don't think it's a huge issue, because a the potential workaround is just explicitly tagging everything with "API". I'll double check old issues and emails to see if I can find anything on this. 

(I also haven't tested this locally so I could be completely misunderstanding what's going on here haha)

> also small nit, since these are case-insensitive regex, I normally keep everything lowercase for consistency

Reply:
Yeah - I figured this was consistent with what we were going for with flat-by-default. Untagged would just show at the root. We can re-nest if you prefer.

FWIW - the existing behavior for the Swagger UI does group under 'default'

![image](https://user-images.githubusercontent.com/10419222/146684989-4f5c8e8f-0680-4e11-a15a-05a9985ad55a.png)

Reply:
Oh okay, yea, if it's not too difficult I would prefer to keep the same behavior. I normally tag everything so it doesn't effect me, but I don't want to speak for everyone else using this. If a flat structure is something you or others need, we can introduce an option (maybe `untaggedSection` defaults to `"API"` could be set to any other string to change the name or false to disable or something?)

On the call what I wanted to avoid was something like this happening:
```
> Cloud Object Storage
  > API
    - Item 1
    - Item 2
    - Item 3
```